src/uct/cuda/base/cuda_md.c

```diff
@@ -85,11 +85,12 @@ UCS_PROFILE_FUNC(ucs_status_t, uct_cuda_base_mem_query,
     uint32_t is_managed        = 0;
     unsigned value             = 1;
     CUdevice cuda_device       = -1;
+    void *base_address         = (void*)address;
```

Comment:
no need to cast

Reply:
Needed this cast from `const void*` to `void*`.